### PR TITLE
css fixes, package script added

### PR DESCRIPTION
**Pull request by yamini2697**
css fixes....

package scripts added

### CODE REVIEW

File: backend/data.js [hunk 1]
```diff
@@ -30,11 +30,9 @@ function readFile() {
 
 }
 var response=readFile();
```
Comment (yaswanthkoneri) [inline backend/data.js]: Why readFile here ? @yamini2697

File: backend/data.js [hunk 2]
```diff
@@ -30,11 +30,9 @@ function readFile() {
...
-console.log(response)
 
             app.get('/employee',function (req,res) {
                 let respo = JSON.parse(readFile());
-                console.log(respo)
                 res.json(respo);
                 //  let remove = req.query.remove;
```
Comment (yaswanthkoneri) [inline backend/data.js]: remove comments and whitespaces

File: task/src/App.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 .flex-container {
   display: flex;
-  background: silver;
+  background: lightgray;
   
 }
 .secondhead{
```
Comment (yaswanthkoneri) [inline task/src/App.css:7]: Read about how to write class names in CSS

**Comment by yaswanthkoneri** [comment]
Change the backend/data.js file name to backend/index.js

File: backend/data.js (hunk 1 above)
Comment (yamini2697) [reply backend/data.js]: It is not required. Just for testing purpose I have written it.

**Pull request merged**